model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/stencils/fused_velocity_advection_stencil_8_to_13.py

```diff
@@ -57,7 +57,7 @@ def _fused_velocity_advection_stencil_8_to_13_predictor(
 
     w_concorr_c = where(
         nflatlev + 1 <= k < nlev,
-        _interpolate_to_half_levels_vp(interpolant=z_w_concorr_mc, wgtfac_c=wgtfac_c),
+        _interpolate_to_half_levels_vp(wgtfac_c=wgtfac_c, interpolant=z_w_concorr_mc),
```

Comment:
does this actually make a difference? In principle it shouldn't so if it does perhaps we need to consider edits on the gt4py side

Reply:
No, it shouldn't make a difference, I just adjusted it to the parameter order